src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs

```diff
@@ -121,5 +121,7 @@ private ISonarLintDaemon Daemon
                 return this.daemon;
             }
         }
+
+
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1186)